DISCOVER/_static/languages.json

```diff
@@ -0,0 +1,28 @@
+[
+    {
+        "language": "en",
+        "name": "English",
+        "preferred": true,
```

Comment:
I would not add a "preferred" key in the metadata, I don't think it makes much sense for languages. What I would add at some point is a key to set languages as "hidden"/not active (see other comment about issue/discussion). That is, we build the documentation for them but we don't add them to the switcher, so we'd filter the hidden ones in `conf.py` before passing that to `html_context`